gadget/validate.go

```diff
@@ -173,46 +181,48 @@ func validateReservedLabels(vs *VolumeStructure, reservedLabels []string) error
 	return nil
 }
 
-func ensureRolesConsistency(roles map[string]*roleInstance, expectedSeed bool) error {
+func ensureRolesConsistency(roles map[string]*roleInstance, hasModes bool) error {
```

Comment:
I don't understand why change this argument name, when we have a totally different function for the classic with modes case?